README.md

```diff
@@ -1,6 +1,12 @@
-# Account service API
+# [DEPRECATED] Account service API
 
------
+## Deprecation Notice
+
+The dfsp-account service was deprecated as of January 2018. This service is no longer maintained as it is no longer in the scope of the Mojaloop OSS community.
```

Comment:
same as the other two

Reply:
please add "is no longer referenced by any releases or in scope of the Mojaloop..."